Service/Request/RequestFactory.php

```diff
@@ -162,9 +163,20 @@ private function getRequestUri(ServiceRequestInterface $serviceRequest): UriInte
         return $this->uriFactory->createUri($baseUrl.$path);
     }
 
+    private function getGetterMethodName(string $property): string
+    {
+        return 'get'.preg_replace_callback(
```

Comment:
I think regexp is overkill here. The current approach (i.e. `str_replace(['-', '_'], '', ucwords($case, '-_'))`) will work as good but will be faster.

Also, the current approach supports double-underscores (although I don't know why would anyone use them) unlike the regexp.